libparsec/crates/protocol/schema/invited_cmds/invite_info.json5

```diff
@@ -26,40 +26,31 @@
                                 "type": "String"
                             },
                             {
-                                // TODO: merge into `created_by`
-                                "name": "greeter_user_id",
-                                "type": "UserID"
+                                "name": "created_by",
+                                "type": "InvitationCreatedBy"
                             },
                             {
-                                // TODO: merge into `created_by`
-                                "name": "greeter_human_handle",
-                                "type": "HumanHandle"
+                                "name": "administrators",
+                                "type": "List<UserGreetingAdministrator>"
```

Comment:
What happens when the list of administrators has changed between the time when the invitation was created and when it is used ?

Edit: This is generated each time invite_info is called right?

Reply:
> This is generated each time invite_info is called right?

Yes. If the claimer picks an administrator that got revoked right after `invite_info` returned, the subsequent `start_greeting_attempt` command will fail with a `GreeterNotAllowed` status. So the claimer can simply restart the process and get a new administrator list from a new `invite_info` command